openedx/adg/common/mailchimp_pipeline/helpers.py

```diff
@@ -22,4 +20,4 @@ def get_enrollment_course_names_and_short_ids_by_user(user):
         return '', ''
 
     short_ids, display_names = zip(*enrollments)
-    return ','.join(map(str, short_ids)), ','.join(display_names)
+    return ', '.join(map(str, short_ids)), ', '.join(display_names)
```

Comment:
Lets revert this change, we explicitly removed the spaces

Reply:
A test was failing due to this. I will revert this and will change the test. It was an inconsistency between test and this method.

Reply:
ok.